.rubocop.yml

```diff
@@ -23,6 +23,7 @@ Metrics/MethodLength:
     - lib/hydra/works/models/concerns/generic_file/virus_check.rb
 
 Metrics/AbcSize:
+  Enabled: false
```

Comment:
Instead of disabling it altogether, can you add file-specific excludes below?  When you remove this line and run rubocop, what does the output look like?  Feel free to gist that and ping me.